composite/settings.gradle.kts

```diff
@@ -1,16 +1,5 @@
 rootProject.name = "apollo-composite"
 
-// Samples
```

Comment:
What is the reason to remove samples from composite project? Are we going to replace this with something else?

Reply:
Definitively an oversight, I only meant to delete the java sample, I'll add the kotlin samples back 👍